corehq/apps/cloudcare/views.py

```diff
@@ -383,7 +384,7 @@ def filter_cases(request, domain, app_id, module_id, parent_id=None):
             ids_only=True,
         )]
 
-    cases = [CommCareCase.wrap(doc) for doc in iter_docs(CommCareCase.get_db(), case_ids)]
+    cases = list(accessor.iter_cases(case_ids))
```

Comment:
making it a list seems to defeat the point of using `iter_cases` could just use `get_cases`